ui/src/app/views/dataset/component.tsx

```diff
@@ -31,17 +40,42 @@ export const styles = {
   }
 };
 
+const ResourceListItem: React.FunctionComponent <
+  {conceptSet: ConceptSet, openConfirmDelete: Function, edit: Function}
+  > = ({conceptSet, openConfirmDelete, edit}) => {
+    return<div style={{border: '0.5px solid #C3C3C3', margin: '.4rem',
+      height: '1.5rem', display: 'flex'}}>
+      <div style={{width: '.75rem', paddingTop: 5, paddingLeft: 10}}>
+        <ResourceCardMenu disabled={false}
```

Comment:
On the mocks, concept sets appear not to have the dropdown menu. That said, I think they probably should? We should check with Lou to confirm here.

Reply:
Yeah I wasn't sure, it looked like _some_ of them had it, and it does say on the card to include it...